src/main/java/ArticleSwitchPage.java

```diff
@@ -0,0 +1,18 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class ArticleSwitchPage {
+
+    WebDriver driver;
+
+    public ArticleSwitchPage(WebDriver driver){ this.driver = driver;}
+
+
+
+    public String isNameExist(){
```

Comment:
the name of the function does not correspond with the action that you are doing. Why not name the function `getProductTitle()` instead?